### PR TITLE
✨Source Glassfrog : Migrate Python CDK to Low-code CDK

### LINKED ISSUE #29281: Glassfrog
https://docs.airbyte.com/integrations/sources/glassfrog

### PR DESCRIPTION
**Pull request by avirajsingh7**
## What
Migrating Source Glassfrog to Low-Code CDK
Closes #29281 

## How
Developed using (Configuration Based Source) low-code CDK

## Recommended reading order

1. manifest.yaml
2. schemas/*

## 🚨 User Impact 🚨

*Are there any breaking changes? What is the end result perceived by the user?*
Yes, there are breaking changes due to Changes to Stream Schema
At many places previous data type were wrong and added some new properties in schema as per response from API

 ##  Test
<details>
<summary>Acceptance Test</summary>
Screenshot 2023-08-10 at 1 24 51 AM

</details>

<details>
<summary>Migrate Connector : Source FreshServices</summary>

- [x] **Community Member? Grant Edit Access to Maintainers [instructions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)**
- [x] **Secrets in the Connector's Spec**
- [x] **Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run ./gradlew :airbyte-integrations:connectors:<name>:integrationTest.**
- [x] **Code Reviews Completed**
  - [x] **Documentation Updated**
  - [x] **Connector's README.md**
  - [x] **Connector's bootstrap.md**
  - [x] **docs/integrations/<source or destination>/<name>.md (including changelog. See changelog example)**
  - [x] **docs/integrations/README.md**
  - [x] **airbyte-integrations/builds.md**
- [x] **PR Name follows PR Naming Conventions**
</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Hello 👋 , right now Airbyte's integration tests are failing for this connector and I'm going to work during the week to fix them. Due a high number of contributions we have a backlog and your contribution will be review next week. We ask to you share the integration test from your side whatever possible and if you don't have credentials to run tests to get in touch with @RealChrisSean as soon as possible. Also some common missing steps contributors does migrating to low code are (and you can double check to make sure you implemented this): 
* adding paginator
* update the documentation changelog + metadata file and dockerfile version
* if the connector has `start_date` parameter you must implement incremental syncs

**Comment by sh4sh** [comment]
Hey @avirajsingh7 could you share the the integration test results here?

**Comment by avirajsingh7** [comment]
> Hey @avirajsingh7 could you share the the integration test results here?
@sh4sh  Already shared them above [here](https://user-images.githubusercontent.com/100823015/259532861-d835abab-af64-4827-9a6a-db672f5e8724.png)
should I run and share them again?

File: airbyte-integrations/connectors/source-glassfrog/Dockerfile [hunk 1]
```diff
@@ -35,4 +35,5 @@ ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
 LABEL io.airbyte.version=0.1.1
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-glassfrog/Dockerfile:37]:
```suggestion
LABEL io.airbyte.version=1.0.0
```

File: airbyte-integrations/connectors/source-glassfrog/metadata.yaml [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
   connectorSubtype: api
   connectorType: source
   definitionId: cf8ff320-6272-4faa-89e6-4402dc17e5d5
-  dockerImageTag: 0.1.1
+  dockerImageTag: 1.0.0
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-glassfrog/metadata.yaml:13]:
```suggestion
  dockerImageTag: 0.2.0
```

File: airbyte-integrations/connectors/source-glassfrog/Dockerfile [hunk 3]
```diff
@@ -34,5 +34,6 @@ COPY source_glassfrog ./source_glassfrog
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.1
+LABEL io.airbyte.version=1.0.0
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-glassfrog/Dockerfile:37]:
```suggestion
LABEL io.airbyte.version=0.2.0
```

File: docs/integrations/sources/glassfrog.md [hunk 4]
```diff
@@ -48,6 +48,7 @@ This Source is capable of syncing the following Streams:
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 1.0.0 | 2023-08-10 | [29306](https://github.com/airbytehq/airbyte/pull/29306) | Migrated to LowCode CDK |
```
Comment (sh4sh) [inline docs/integrations/sources/glassfrog.md:51]:
```suggestion
| 0.2.0 | 2023-08-10 | [29306](https://github.com/airbytehq/airbyte/pull/29306) | Migrated to LowCode CDK |
```

File: airbyte-integrations/connectors/source-glassfrog/metadata.yaml [hunk 5]
```diff
@@ -1,22 +1,27 @@
 data:
+  allowedHosts:
+    hosts:
+      - "*" # Please change to the hostname of the source.
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-glassfrog/metadata.yaml:4]:
```suggestion
      - api.glassfrog.com
```

**Review by sh4sh — APPROVED** [review]

File: airbyte-integrations/connectors/source-glassfrog/metadata.yaml [hunk 6]
```diff
@@ -1,22 +1,27 @@
...
   license: MIT
   name: Glassfrog
-  registries:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: "2022-06-16"
   releaseStage: alpha
+  supportLevel: community
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-glassfrog/metadata.yaml:21]:
```suggestion
```

**Comment by sh4sh** [comment]
Thank you for your contribution!

**Pull request merged**